src/makefile

```diff
@@ -3,14 +3,14 @@ EXECUTABLES = sed rm mkdir date zip m4 cp cat
 K := $(foreach exec,$(EXECUTABLES),\
 	$(if $(shell which $(exec)),some string,$(error "Could not find the dependency '$(exec)'. If you are unable to install it, there are complete builds of Automail at https://github.com/hohMiyazawa/Automail/releases or https://greasyfork.org/en/scripts/370473-automail")))
 
-all: pre-build build build/automail.js build/boneless.js build/firefox_addon.zip post-build
+all: pre-build build build/automail.user.js build/boneless.js build/firefox_addon.zip post-build
 
 pre-build:
 	rm -f build/userModules.js
 
-build/automail.js: automail.m4 settings.js alias.js css/global.css conditionalStyles.js utilities.js purify.js graphql.js localisation.js controller.js build/userModules.js HOWTO.js
-	m4 --prefix-builtins automail.m4 > build/automail.js
-	date +"%s" | sed 's_^_//Automail built at _' >> build/automail.js
+build/automail.user.js: automail.m4 settings.js alias.js css/global.css conditionalStyles.js utilities.js purify.js graphql.js localisation.js controller.js build/userModules.js HOWTO.js
```

Comment:
the only thing you wanna check on greasy fork, just in case it breaks something is this
@hohMiyazawa

Reply:
Tested. The renaming does not affect the userscript upload procedure.